### DISCUSSION TITLE
Help with configuring RTL app with remote lnd

### DISCUSSION THREAD

**Discussion by ray-bun** [Ride-The-Lightning/RTL#1403] [General]
Hi, I've installed RTL on a seperate container. How do I configure RTL to use lnd that is on a seperate server.

**Comment by ShahanaFarooqui** [comment]
- You can configure lnServerUrl (env LN_SERVER_URL) with the LND's IP and rest port.
- You will need LND's admin.macaroon mounted/copied on RTL container. It should then be configured with macaroonPath (env MACAROON_PATH) value.
- Make sure that LND's IP and port are accessible/exposed.

**Reply by ray-bun** [reply]
Thanks. Do you have a sample config please? How about the lnd.conf ?
Are you suggesting that I mount the LND remote server path onto the RTL container?

**Reply by ShahanaFarooqui** [reply]
RTL sample config can be found [here](https://github.com/Ride-The-Lightning/RTL/blob/master/Sample-RTL-Config.json). lnd.conf will not have any impact on RTL as long as it is working and IP:port are accessible. 

No, path can not be mounted. I am suggesting to mount the admin.macaroon OR copy the admin.macaroon somewhere in RTL container. Because RTL will need this admin.macaroon for LND authentication.

Then configure the lnServerUrl value with LND's REST url.

You can add RTL's configurations via Docker's [environment variables](https://github.com/Ride-The-Lightning/RTL/blob/master/.github/docs/Application_configurations.md) ### Environment variables or via RTL-Config.json file.

**Reply by ray-bun** [reply]
Thank you
